app/src/main/java/com/nextcloud/talk/jobs/ShareOperationWorker.java

```diff
@@ -70,12 +75,16 @@ public ShareOperationWorker(@NonNull Context context, @NonNull WorkerParameters
     @NonNull
     @Override
     public Result doWork() {
+
+        String metaData = "{\"messageType\":\"voice-message\"}";
+
         for (int i = 0; i < filesArray.size(); i++) {
             ncApi.createRemoteShare(credentials,
                     ApiUtils.getSharingUrl(baseUrl),
                     filesArray.get(i),
                     roomToken,
-                    "10")
+                    "10",
+                     metaData)
```

Comment:
This is only done for voice-messages, right? Or are you flagging all as such?

Reply:
right this was just for testing. changed this now that only voice messages are flagged